examples/vertx/README.md

```diff
@@ -0,0 +1,11 @@
+# Containerize a [Eclipse Vert.x](https://vertx.io/) application with Jib
+
+This is an example of how to easily build a Docker image for a [Eclipse Vert.x application](https://vertx.io/) with Jib.
+
+```shell
+/gradlew jibDockerBuild
+
+docker run -d --rm -p 8080:8080 vertx-jib-example
+```
+
```

Comment:
Could you include a _More Information_ section with links to both Jib (for people who come directly into this page from a search) and to Vert.x?

Reply:
Sure!